### ISSUE TITLE
Debian GNU/Linux and Ubuntu Install Directions not working

### ISSUE THREAD

**Issue by kutrumbo** [apache/arrow#34677]
### Describe the bug, including details regarding any error messages, version, and platform.

I started getting build failures for our application on the Arrow install step as of a few hours ago (2023-03-22 03:26:31)

The build is done on an Ubuntu server and follows the directions described here (and this build had been working fine previously): https://arrow.apache.org/install/#c-and-glib-c-packages-for-debian-gnulinux-ubuntu-almalinux-centos-and-amazon-linux

The issue looks to be that the file being returned from the following step is not a Debian package file and instead is just returning html:
```
wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
```

The specific ubuntu version where I am seeing this is Ubuntu Jammy - the URL being: https://apache.jfrog.io/artifactory/arrow/ubuntu/apache-arrow-apt-source-latest-jammy.deb

It looks like requests to that URL are being redirected to https://landing.jfrog.com/reactivate-server/apache

```
# this is the output if I curl it with -I
curl -I https://apache.jfrog.io/artifactory/arrow/ubuntu/apache-arrow-apt-source-latest-jammy.deb
HTTP/1.1 302 Moved Temporarily
Date: Wed, 22 Mar 2023 05:53:58 GMT
Content-Type: text/html
Content-Length: 138
Connection: keep-alive
Location: https://landing.jfrog.com/reactivate-server/apache
```

### Component(s)

Documentation, Release

**Comment by assignUser** [comment]
Duplicate of #34675 

Sorry for the inconvenience, the apache jfrog instance is down at the moment. We are waiting on feedback from the ASF infra team.
